build.info

```diff
@@ -110,9 +110,9 @@ ENDIF
 
 # This file sets the build directory up for CMake inclusion
 GENERATE[OpenSSLConfig.cmake]=exporters/cmake/OpenSSLConfig.cmake.in
-DEPEND[OpenSSLConfig.cmake]=builddata.pm
+DEPEND[OpenSSLConfig.cmake]=installdata.pm
 GENERATE[OpenSSLConfigVersion.cmake]=exporters/cmake/OpenSSLConfigVersion.cmake.in
-DEPEND[OpenSSLConfigVersion.cmake]=builddata.pm
+DEPEND[OpenSSLConfigVersion.cmake]=installdata.pm
```

Comment:
Nope.

There are two variants of OpenSSLConfig.cmake / OpenSSLConfigVersion.cmake, on built in the top directory, and one built in the subdirectory `exporters/`.  They are intended to produce what CMake's `export` and `install(EXPORT)` do, respectively.
The first is intended for those who want to use a build of OpenSSL directly, while the second (those in `exporters/`) is intended for installation, i.e. to end up among installed CMake modules.

Therefore, those one is built using `builddata.pm`, while you will see `installdata.pm` being used in `exporters/build.info`.

Reply:
Maybe the comment on line 111 could be enhanced.  If you don't know CMake, I can see that it's a bit terse...